src/main/java/randoop/util/ClassFileConstants.java

```diff
@@ -629,47 +678,117 @@ public static ConstantSet getConstants(String classname, ConstantSet result) {
       }
       for (Integer x : cs.ints) {
         try {
-          map.add(clazz, new NonreceiverTerm(JavaTypes.INT_TYPE, x));
+          int frequency = getFrequencyOfTerm(x, cs);
+          map.add(clazz, new NonreceiverTerm(JavaTypes.INT_TYPE, x, frequency));
         } catch (IllegalArgumentException e) {
-          System.out.println("Ignoring int constant value: " + e.getMessage());
+          throw new BugInRandoopException(e);
         }
       }
       for (Long x : cs.longs) {
         try {
-          map.add(clazz, new NonreceiverTerm(JavaTypes.LONG_TYPE, x));
+          int frequency = getFrequencyOfTerm(x, cs);
+          map.add(clazz, new NonreceiverTerm(JavaTypes.LONG_TYPE, x, frequency));
         } catch (IllegalArgumentException e) {
-          System.out.println("Ignoring long constant value: " + e.getMessage());
+          throw new BugInRandoopException(e);
         }
       }
       for (Float x : cs.floats) {
         try {
-          map.add(clazz, new NonreceiverTerm(JavaTypes.FLOAT_TYPE, x));
+          int frequency = getFrequencyOfTerm(x, cs);
+          map.add(clazz, new NonreceiverTerm(JavaTypes.FLOAT_TYPE, x, frequency));
         } catch (IllegalArgumentException e) {
-          System.out.println("Ignoring float constant value: " + e.getMessage());
+          throw new BugInRandoopException(e);
         }
       }
       for (Double x : cs.doubles) {
         try {
-          map.add(clazz, new NonreceiverTerm(JavaTypes.DOUBLE_TYPE, x));
+          int frequency = getFrequencyOfTerm(x, cs);
+          map.add(clazz, new NonreceiverTerm(JavaTypes.DOUBLE_TYPE, x, frequency));
         } catch (IllegalArgumentException e) {
-          System.out.println("Ignoring double constant value: " + e.getMessage());
+          throw new BugInRandoopException(e);
         }
       }
       for (String x : cs.strings) {
         try {
-          map.add(clazz, new NonreceiverTerm(JavaTypes.STRING_TYPE, x));
+          int frequency = getFrequencyOfTerm(x, cs);
+          map.add(clazz, new NonreceiverTerm(JavaTypes.STRING_TYPE, x, frequency));
         } catch (IllegalArgumentException e) {
-          System.out.println("Ignoring String constant value: " + e.getMessage());
+          throw new BugInRandoopException(e);
         }
       }
       for (Class<?> x : cs.classes) {
         try {
-          map.add(clazz, new NonreceiverTerm(JavaTypes.CLASS_TYPE, x));
+          int frequency = getFrequencyOfTerm(x, cs);
+          map.add(clazz, new NonreceiverTerm(JavaTypes.CLASS_TYPE, x, frequency));
         } catch (IllegalArgumentException e) {
-          System.out.println("Ignoring Class<?> constant value: " + e.getMessage());
+          throw new BugInRandoopException(e);
         }
       }
     }
     return map;
   }
+
+  /**
+   * Increment value in map. If the key is null, we ignore this call and return.
```

Comment:
Under what circumstances is the key null?  I wouldn't expect this to happen.
If it does not, you could use `CollectionsPlume.incrementMap` instead.